src/IntegrationUtils.ts

```diff
@@ -289,20 +289,13 @@ function getSelectedSecret(): Promise<string | undefined> {
 
 function getSelectedResource(): Promise<string> {
 	return new Promise<string> ( async (resolve, reject) => {
-		let returnedResources: string = '';
-		const fileUri = await vscode.window.showOpenDialog(ResourceOptions);
-		if (fileUri === undefined || fileUri.length === 0) {
-			reject(new Error("No Resource file(s) specified."));
+		const fileUris = await vscode.window.showOpenDialog(ResourceOptions);
+		if (fileUris === undefined || fileUris.length === 0) {
+			reject(new Error('No Resource file(s) specified.'));
+		} else if(fileUris.length >= 2) {
```

Comment:
Why do you use `>=2` instead of `>1`?

Reply:
I found it clearer that the case of `1` was not part of any of the written conditions by not writing it.